### PR TITLE
Add ipv6 support

### PR DESCRIPTION
**Pull request by AMecea**

### CODE REVIEW

File: django_project/settings/base.py [hunk 1]
```diff
@@ -240,7 +240,7 @@
     },
     'update_ns_propagated': {
         'task': 'zinc.tasks.update_ns_propagated',
-        'schedule': 60
+        'schedule': 60 * 10,
```
Comment (bogdanpetrea) [inline django_project/settings/base.py:243]: Why's that?

File: tests/dns/test_policy_record_tree.py [hunk 2]
```diff
@@ -25,7 +25,21 @@ def strip_ns_and_soa(records, zone_root):
...
+
+    has_ip_of_type = False
+    for pm in policy_members:
+        if region and pm.region != region:
+            continue
+
+        if sep in pm.ip.ip:
+            has_ip_of_type = True
+
+    return has_ip_of_type
```
Comment (bogdanpetrea) [inline tests/dns/test_policy_record_tree.py:39]: nitpick
```suggestion
            return True

    return False
```

**Review by bogdanpetrea — CHANGES_REQUESTED** [review]
Please squash the migrations

File: django_project/settings/base.py (hunk 1 above)
Comment (AMecea) [reply django_project/settings/base.py:243]: The NS propagated task checks if the actual NS used by us are set or not. This helps in fast debuging, it's not an important task, based on this info there are not taken any decisisons.
So I increased it to 10 minutes.

I will use `timedelta`.

**Review by bogdanpetrea — APPROVED** [review]

**Pull request merged**